Libraries/xcsdk/CMakeLists.txt

```diff
@@ -23,3 +23,4 @@ target_include_directories(xcsdk PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/Headers")
 add_executable(xcrun Tools/xcrun.cpp)
 target_link_libraries(xcrun xcsdk util)
 
+install(TARGETS xcrun RUNTIME DESTINATION bin)
```

Comment:
Could you move this to a separate diff? Might be good to add `install()` calls for all the utilities together.

I also wonder if this would need `install()` calls for the shared libraries too? I'm not sure how CMake handles that with install names on OS X.